examples/getting_started/run.py

```diff
@@ -12,7 +12,96 @@
+        assert "wheel_encoder" in obs.keys()
+        self._wheel_rotation_speeds = obs["wheel_encoder"]
+
+        return obs
+
+    def _apply_action(self, action):
+
+        assert "set_duty_cycle" in action.keys()
+        assert action["set_duty_cycle"].shape[0] == 2
+        assert self._wheel_rotation_speeds is not None
+
+        duty_cycles = np.array([action["set_duty_cycle"][0], action["set_duty_cycle"][1], action["set_duty_cycle"][0], action["set_duty_cycle"][1]], dtype=np.float64)
+
+        motor_velocity_constant = self._config.OPENBOT_ENV.MOTOR_VELOCITY_CONSTANT # Motor torque constant in [N.m/A]
+        gear_ratio              = self._config.OPENBOT_ENV.GEAR_RATIO              # Gear ratio of the OpenBot motors
+        motor_torque_constant   = 1.0 / motor_velocity_constant                    # Motor torque constant in [rad/s/V]
+
+        battery_voltage         = self._config.OPENBOT_ENV.BATTERY_VOLTAGE         # Voltage of the battery powering the OpenBot [V]
+        control_dead_zone       = self._config.OPENBOT_ENV.CONTROL_DEAD_ZONE       # Absolute duty cycle (in the ACTION_SCALE range) below which a command does not produce any torque on the vehicle
+        motor_torque_max        = self._config.OPENBOT_ENV.MOTOR_TORQUE_MAX        # Motor maximal torque [N.m]
+        electrical_resistance   = self._config.OPENBOT_ENV.ELECTRICAL_RESISTANCE   # Motor winding electrical resistance [Ohms]
+
+        # Speed multiplier defined in the OpenBot to map a [-1,1] action to a suitable command to 
+        # be processed by the low-level microcontroller. For more details, feel free to check the
+        # "speedMultiplier" command in the OpenBot code.
+        # https://github.com/isl-org/OpenBot/blob/master/android/app/src/main/java/org/openbot/vehicle/Vehicle.java#L375
+        action_scale = self._config.OPENBOT_ENV.ACTION_SCALE
+
+        # Acquire the ground truth motor and wheel velocity for motor counter-electromotive force
+        # computation purposes (or alternatively friction computation purposes).
+
+        # The ground truth velocity of the robot wheels in [rad/s]
+        wheel_rotation_speeds = self._wheel_rotation_speeds
+
+        # The ground truth rotation speed of the motors in [rad/s]
+        motor_speeds = gear_ratio * wheel_rotation_speeds
+
+        # Compute the counter electromotive force using the motor torque constant
+        counter_electromotive_forces = motor_torque_constant * motor_speeds # Expressed in [V]
+
+        # The electrical current allowed to circulate in the motor is the result of the
+        # difference between the applied voltage and the counter electromotive force
+        motor_winding_currents = ((battery_voltage * duty_cycles) - counter_electromotive_forces) / electrical_resistance # Expressed in [A]
+
+        # The torque is then obtained using the torque coefficient of the motor in [N.m]
+        drive_torques = motor_torque_constant * motor_winding_currents
```

Comment:
I think you misunderstood my comment, or my comment had a typo in it. This (line 86) should be called `motor_torques`

Reply:
Update: my comment had a typo in it. I gave the wrong guidance. My bad.